src/main/scala/io/github/jchapuis/fs2/kafka/mock/impl/NativeMockKafkaProducer.scala

```diff
@@ -76,6 +76,18 @@ private[mock] class NativeMockKafkaProducer(
       else IO.pure(false)
   ).map { case (_, value) => value }
 
+  def nextEventualValueOrRedactionFor[K: Eq, V](topic: String, key: K)(implicit
```

Comment:
i notice usage of the term "Redaction" here vs. "Redacted" elsewhere, is it intentional?

Reply:
yeah, I was thinking "produced" / "a production" == "redacted" / "a redaction" (== "evaluated" / "a value")